core/src/dird/ua_output.cc

```diff
@@ -1617,22 +1617,37 @@ bool bsendmsg(void* ctx, const char* fmt, ...)
  * The following UA methods are mainly intended for GUI
  * programs
  */
+
+void UaContext::vSendMsg(int signal,
+                         const char* messagetype,
+                         const char* fmt,
+                         va_list arg_ptr)
+{
+  /* send current buffer */
+  PoolMem message;
+  send->SendBuffer();
+  if (signal) {
+    BareosSocket* bs = UA_sock;
+    if (bs && api) bs->signal(signal);
```

Comment:
no need for a pointer alias.
```suggestion
    if (UA_sock && api) UA_sock->signal(signal);
```

Reply:
Good point